### ISSUE TITLE
Allow configuring aeron properties via application.conf

### ISSUE THREAD

**Issue by ktoso** [akka/akka#23889]
Some of the aeron settings are likely to be always needed to be tweaked, it's nicer if we can put that configuration in application.conf, pick them up at startup and set the env variables for aeron to pick up (or pass it a config?).

**Comment by ktoso** [comment]
The usual things to tweak are: https://github.com/real-logic/aeron/wiki/Performance-Testing#throughput-testing

We could provide some "human readable recommended" values so that we'd `aeron.mtu.length=internal-network` etc?

**Comment by guidomedina** [comment]
I agree with Patrik here, having the shared driver inside Akka IMHO will be the exceptional case because of the amount of shared memory each driver instance requires, if the user still wants the driver inside Akka maybe the right solution is document key recommended options but always point the user to Aeron config section.

As an example one of my projects is running on an Akka cluster of 15 JVMs, 3 JVMs per physical server and each physical server has one dedicated Aeron driver so all Akka nodes are running as client.

On an IDE like IntelliJ is the same, because projects having Akka remote with Artery will require one driver, IMHO it is better that the user know this and make the effort of bringing up his own driver for his own "later" good.
